src/utils.tsx

```diff
@@ -235,7 +235,7 @@ export function setCaretPosition(el: HTMLInputElement, caretPos: number) {
     }
 
     // fail city, fortunately this never happens (as far as I've tested) :)
-    el.focus();
+    el.focus?.();
```

Comment:
Q: What is the element type coming here. This function expects the element type to be input. Can you share more details on how the custom input is implemented.